orchagent/response_publisher.cpp

```diff
@@ -90,7 +90,10 @@ void RecordResponse(const std::string &response_channel, const std::string &key,
 
 } // namespace
 
-ResponsePublisher::ResponsePublisher() : m_db("APPL_STATE_DB", 0)
+ResponsePublisher::ResponsePublisher(bool buffered) :
+    m_db(std::make_unique<swss::DBConnector>("APPL_STATE_DB", 0)),
+    m_pipe(std::make_unique<swss::RedisPipeline>(m_db.get())),
```

Comment:
>m_db [](http://example.com/codeflow?start=49&length=4)

You are assuming m_pipe initialized after m_db. The sequence is tricky. Suggest move m_pipe to function body.

Reply:
@qiluo-msft Same reply https://github.com/sonic-net/sonic-swss-common/pull/708#discussion_r1070343699